### ISSUE TITLE
Lua error on "Backspace" in prompt: Error executing lua [string ":lua"]:1: module 'guihua.ListViewCtrl' not found:

### ISSUE THREAD

**Issue by RobertBuhren** [ray-x/navigator.lua#35]
### Issue
Pressing "Backspace" to delete a character in the document symbol window results in a lua error:
```
E5108: Error executing lua [string ":lua"]:1: module 'guihua.ListViewCtrl' not found:                                                                                                
        no field package.preload['guihua.ListViewCtrl']
        no file './guihua/ListViewCtrl.lua'
        no file '/usr/share/luajit-2.0.5/guihua/ListViewCtrl.lua'
        no file '/usr/local/share/lua/5.1/guihua/ListViewCtrl.lua'
        no file '/usr/local/share/lua/5.1/guihua/ListViewCtrl/init.lua'
        no file '/usr/share/lua/5.1/guihua/ListViewCtrl.lua'
        no file '/usr/share/lua/5.1/guihua/ListViewCtrl/init.lua'
        no file './guihua/ListViewCtrl.so'
        no file '/usr/local/lib/lua/5.1/guihua/ListViewCtrl.so'
        no file '/usr/lib/lua/5.1/guihua/ListViewCtrl.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
        no file './guihua.so'
        no file '/usr/local/lib/lua/5.1/guihua.so'
        no file '/usr/lib/lua/5.1/guihua.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
```

### Screenshot
Result of pressing "Backspace" in the document symbols (g0) window.
![Screenshot from 2021-07-19 10-04-24](https://user-images.githubusercontent.com/815756/126125277-bbaa402a-0245-4006-aa6c-6911c6daca0e.png)

### Setup

Tested with this minimal config and neovim-0.5 on archlinux. Language server is clangd.
```
set termguicolors
call plug#begin('~/.local/share/nvim/plugged')
Plug 'neovim/nvim-lspconfig'
Plug 'nvim-treesitter/nvim-treesitter'

Plug 'ray-x/guihua.lua'
Plug 'ray-x/navigator.lua'

call plug#end()

lua <<EOF
local single = {"╭", "─", "╮", "│", "╯", "─", "╰", "│"}
require"navigator".setup({
  debug = true,
  width = 0.7,
  border = single, -- "single"
})
EOF
set mouse=a
```

**Comment by ray-x** [comment]
Thanks for your bug report.
Please update plugin `guihua.lua`

I had not test linux platform before and it seems the lua JIT behaviour differently.

**Comment by RobertBuhren** [comment]
Ok, backspace kinda works now, however not as expected:

I can delete a character using backspace, but the search result is not updated until I type a new character. 
Also when I select an entry using "Enter", I get the following error: 
```
E5108: Error executing lua .../share/nvim/plugged/navigator.lua/lua/navigator/util.lua:275: Vim(edit):E32: No file name
```

**Comment by ray-x** [comment]
It is a bug. But I do not have archlinux ready to debug this issue. May take a few days.

**Comment by ray-x** [comment]
I installed archlinux today and this issue can not been reproduce, it should have been fixed by #5af79c7
